### DISCUSSION TITLE
Detailed API Docs

### DISCUSSION THREAD

**Discussion by paherbst** [OpenDDS/OpenDDS#4821] [General]
Hi, I'm fumbling through trying to use OpenDDS.  I'm referencing the "[Developers Guide](https://opendds.readthedocs.io/en/latest-release/devguide/index.html)" and the "[DevGuideExamples](https://github.com/OpenDDS/OpenDDS/tree/master/DevGuideExamples)".

But I'm finding most of the API isn't covered in these places.  And the headers do not provide comments to give hints to most of the interfaces.

Is there a better source of documentation?

**Comment by jrw972** [comment]
That depends on what you are trying to do.
If you can give some example questions that you are trying to answer, then we can probably point you in the right direction.

**Comment by iguessthislldo** [comment]
I answered basically the same question in https://github.com/OpenDDS/OpenDDS/discussions/4577.
